sdks/java/maven-archetypes/starter/src/main/resources/archetype-resources/src/main/java/StarterPipeline.java

```diff
@@ -49,14 +49,14 @@ public static void main(String[] args) {
         PipelineOptionsFactory.fromArgs(args).withValidation().create());
 
     p.apply(Create.of("Hello", "World"))
-    .apply(ParDo.of(new OldDoFn<String, String>() {
-      @Override
+    .apply(ParDo.of(new DoFn<String, String>() {
```

Comment:
`MapElements` everywhere? I don't know how much about `DoFn` we're trying to communicate in the starter pipeline